android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/FormConfigurationJsonFormActivity.java

```diff
@@ -38,11 +39,22 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        formUtils = new FormUtils();
+        formUtils = getFormUtils();
         JSONObject jsonObject = getmJSONObject();
         checkIfFormUpdate(jsonObject);
     }
 
+    private FormUtils getFormUtils(){
+        if (!this.isUsingLocaleForms()){
+            return new NoLocaleFormUtils();
+        }
+        return new FormUtils();
+    }
+
+    protected boolean isUsingLocaleForms(){
```

Comment:
I thought of a better name `supportsLocaleBasedForms()`? Does it sound similar? Better? Worse

Reply:
Yes, it's better